bedwars-plugin/src/main/java/com/andrei1058/bedwars/commands/bedwars/subcmds/regular/CmdSpect.java

```diff
@@ -0,0 +1,88 @@
+package com.andrei1058.bedwars.commands.bedwars.subcmds.regular;
+
+import com.andrei1058.bedwars.api.arena.GameState;
+import com.andrei1058.bedwars.api.arena.IArena;
+import com.andrei1058.bedwars.api.command.ParentCommand;
+import com.andrei1058.bedwars.api.command.SubCommand;
+import com.andrei1058.bedwars.api.language.Language;
+import com.andrei1058.bedwars.api.language.Messages;
+import com.andrei1058.bedwars.arena.Arena;
+import com.andrei1058.bedwars.configuration.Sounds;
+import org.bukkit.Bukkit;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+import java.util.List;
+
+public class CmdSpect extends SubCommand {
+
+    /**
+     * Create a sub-command for a bedWars command
+     * Make sure you return true or it will say command not found
+     *
+     * @param parent parent command
+     * @param name   sub-command name
+     */
+    public CmdSpect(ParentCommand parent, String name) {
+        super(parent, name);
+        setPriority(21);
+        showInList(true);
+    }
+
+    /**
+     * Add your sub-command code under this method
+     *
+     * @param args command arguments
+     * @param s command executor
+     */
+    @Override
+    public boolean execute(String[] args, CommandSender s) {
+
+        if (!(s instanceof Player)) return true;
+
+        Player player = (Player) s;
+
+        if (args.length != 1) {
+            s.sendMessage(Language.getMsg(player, Messages.COMMAND_SPECT_USAGE));
+            return true;
+        }
+
+        if (Arena.isInArena(player)) return true;
```

Comment:
If you are spectating a game and using this command it should work as /tp to target.